server/server.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	_ "image/gif"
+	_ "image/jpeg"
+	_ "image/png"
+	"math/rand"
+	"net/http"
+
+	"image"
+
+	"sync"
+)
+
+type server struct {
+	image img
+}
+
+type img struct {
+	sync.RWMutex
+	frame image.Image
+	count int
+}
+
+var message = []string{"right", "left"}
```

Comment:
add forward and stop ?

Reply:
plutot forward et backward

Reply:
This is not intended to return the actual command. This is a message example.